src/views/preview/comment/compose-comment.jsx

```diff
@@ -165,9 +166,9 @@ class ComposeComment extends React.Component {
                         <CommentingStatus>
                             <p>Scratch thinks your comment was disrespectful.</p>
                             <p>
-                                For the next {this.state.muteExpiresAt} you
+                                For the next {formatTime.formatTimeUntil(this.state.muteExpiresAt, window._locale)} you
```

Comment:
Revise to "You will be able to comment again {formatTime.formatRelativeTime}" and make this the header.

Reply:
Done. I think -- let me know if I got it right.